rbac/management/audit_log/model.py

```diff
@@ -69,21 +69,13 @@ def get_resource_item(self, r_type, request, *args, **kwargs):
         """Find related information (eg, name, id, etc...) for each resource item."""
         verify_tenant = self.get_tenant_id(request)
         if r_type == AuditLog.ROLE:
-            if request.data != {}:
```

Comment:
Why this is removed? Is it because it won't never happen?

Reply:
Yes! I was thinking that for the deletion/removal aspect, I will need to pull in the object in order to make sure the log the data after it gets deleted or removed. 
The else statement within the get_resource_item() was meant for the deletion process, but since the get_resource_item() would only work for the create/edit/add, for all of these actions, the body cannot be empty, so the else statement would be irrelevant! 
At least, that's what I had in mind!! I'm not entirely sure if that's the case moving forward, but I figured that if I did need it, I would just add it in later PRs! For now, the else statement is irrelevant (from what I understand!)

Reply:
Cool, sounds good